### DISCUSSION TITLE
SMR firefly Reduced Balance

### DISCUSSION THREAD

**Discussion by Jojopogs** [iotaledger/firefly#4023] [Help & Support]
I just updated my firefly to the new version but i noticed that my SMR balance was reduced from 1.7B to just 1.7k SMR. Did anyone has experienced the same issue?

**Comment by HankMoody089** [comment]
https://twitter.com/c_varley/status/1551935073372495874?s=20

**Comment by HankMoody089** [comment]
Not an issue

**Comment by Mcblazeallot** [comment]
> I just updated my firefly to the new version but i noticed that my SMR balance was reduced from 1.7B to just 1.7k SMR. Did anyone has experienced the same issue?

Yes im having the same thing happen to me went from 1.4B to 1.5K SMR whats up with that???

**Reply by nakkimusic** [reply]
I have also had this reduction in the number of tokens obtained.... do you know anything?

**Reply by no8ody** [reply]
Folks, in the previous versions the XXXX SMR were actually in the Glow unit and that has now been changed to the correct SMR unit.

1 SMR = 1000000 Glow

**Reply by AXMAS64** [reply]
Hello!
Tell you that when we started IOTA staking to get Shimmer it was: 1 shimmer every 10 seconds for each Miota in Staking, no 1 Glow every 10 seconds, they were Shimmer !!!
I can't understand why at first you had to create as many Shimmers as possible to help the ecosystem, because you were going to need them for the development of apps and then the governance decides about the 6 decimal places so that the touch is worth 1,000,000 more, my friend A bad feeling of all these decisions has remained. I feel cheated since they did not do things as they promised us, with the Shimmer governance making the decision for all of us to divide the amount of Shimmer obtained in the staking by one million! They could have booted Firefly's wallet if we wanted to split our Shimmers for a million.
I think that would have been the most sensible and the most fair!

**Reply by MiKeF05** [reply]
That's completely wrong. With the decimal places the tokens have the same value. Since the total supply has also the decimals it stays the same like before. It is only math.

**Reply by nakkimusic** [reply]
ok, mathematics is mathematics, but I think the anger is ...if we think that in the future SMR will have an exchange value (let's suppose 0.001MIOTA), it is not the same to multiply it by the SMR harvested initially in the stakin, than to multiply it by the current SMR/1000000, because I suppose that the exchange value will be per SMR and not per GLOW. I hope I have explained

**Reply by AXMAS64** [reply]
Thank you for your words nakkimusic !!

**Reply by MiKeF05** [reply]
But you need to look at the market cap of SMR. Let´s suppose the market cap of SMR will be 500.000.000$. Then the token price with the old supply of 1.450.896.407.249.092 and without decimals will be $0.0000003446.
If you have 100.000.000 SMR without decimals they have a total value of 34,46$.

Now let´s look at the same market cap with the introduction of decimals. The token price with the old supply of 1.450.890.470,249092 will be 0,3446$. With decimals you now have 100 SMR and they have a total value of 34,46$. So it is the same like before. Or am I wrong?

**Reply by AXMAS64** [reply]
Hello MiKeF05!
Thanks for entering this thread. I think we both know how to count and maybe a bit of math too!!
First of all tell you that one thing is Quantity and another thing is Value.
Ami, as a Shimmer holder, I can care less about the value of the market cap at the launch of Shimmer and I can wait holding for as many years as it takes until I have seen how my Shimmers were gradually gaining value and gradually making those decimals disappear 6 ?
Speaking of mathematics, 15,000,000 units of Shimmer or whatever they are, let's call them Shimmer or whatever you want can never be 15 units, another thing as you very well say is that we are giving 15,000,000 units the value of 15 units .
I have read somewhere that they had reduced it to facilitate the figures so that they are not so long, if that is the case, they could have thought of it before, during development! And having made a participatory botación of all the shimmer holders, like when they have asked us to botte in case we wanted to or not, that more Shimmer would be created for the development of the community.
Your math still doesn't convince me.
I send a cordial greeting !!
